src/metabase/public_settings.clj

```diff
@@ -940,3 +940,10 @@ See [fonts](../configuring-metabase/fonts.md).")
   :export?    false
   :default    true
   :type       :boolean)
+
+(defsetting download-row-limit
+  (deferred-tru "Exports row limit, not including xlsx.")
+  :visibility :internal
+  :export?    false
```

Comment:
is there a reason why this setting is not exported?

Reply:
Nope, just a copy/paste thing. I changed it to true.